### ISSUE TITLE
Markers created with discrete option, does not animate

### ISSUE THREAD

**Issue by egurudutt** [apexcharts/apexcharts.js#477]
## Markers created with discrete option does not animate 

I am trying to get effect as in att
![20190403_210725](https://user-images.githubusercontent.com/11315683/55507413-ea074800-5657-11e9-8b9b-e3fb3479f8bf.gif)

## Explanation
As shown in attached image, after clicking on Graph1 / Graph2 button markers should animate together with graph 
 - But here markers are moved first and then line graph animates to new position.

Options used to draw both graph with discrete marker option are 
Graph1 

> {
"options":{"animations":{"enabled":true,"easing":"easeinout","speed":800,"animateGradually":{"enabled":true,"delay":150},"dynamicAnimation":{"enabled":true,"speed":350}},
"stroke":{"curve":"smooth"},
"title":{"text":"Current Markers","align":"left"},
"grid":{"borderColor":"#e7e7e7","row":{"colors":["#f3f3f3","rgba(0, 0, 0, 0)"],"opacity":0.5}},
"markers":{"size":5,"
discrete":[**{"seriesIndex":0,"dataPointIndex":0,"size":12},{"seriesIndex":0,"dataPointIndex":1,"size":6},{"seriesIndex":0,"dataPointIndex":2,"size":6}**]},
"xaxis":{"categories":[],"title":{"text":"Year"},"type":"numeric","convertedCatToNumeric":true,"labels":{}},
"yaxis":{"title":{"text":"K Point"}}},
"series":[**{"data":[10,20,30,40,50,60]}**]}

For Graph2

> {
"options":{
    "animations":{"enabled":true,"easing":"easeinout","speed":800,
                 "animateGradually":{"enabled":true,"delay":150},
                 "dynamicAnimation":{"enabled":true,"speed":350}},
    "stroke":{"curve":"smooth"},
    "title":{"text":"Current Markers","align":"left"},
    "grid":{"borderColor":"#e7e7e7",
            "row":{"colors":["#f3f3f3","rgba(0, 0, 0, 0)"],"opacity":0.5}},
            "markers":{"size":5,
                        "discrete"**:[{"seriesIndex":0,"dataPointIndex":0,"size":12}, 
                                         {"seriesIndex":0,"dataPointIndex":1,"size":6}, 
                                         {"seriesIndex":0,"dataPointIndex":2,"size":6}]**},
     "xaxis":{"categories":[],
                    "title":{"text":"Year"},
                    "type":"numeric","convertedCatToNumeric":true,"labels":{}},
      "yaxis":{"title":{"text":"K Point"}}},
"series":[**{"data":[0,5,10,15,20,15]}**]}

**Comment by junedchhipa** [comment]
Markers don't animate alongside lines and this is the behavior for normal/discrete markers.
I will check soon why the animation has stopped for all markers while updating charts.

**Comment by egurudutt** [comment]
Thank you :)

Looking forwards to it :)

**Comment by SamLewis0602** [comment]
I'd love for this issue to get some TLC if possible! Having the markers animate with our lines would contribute to a smoother experience for our users. 

Also, just wanted to say thank you for the incredible documentation. I was implementing different charting libraries on a project at work that will have many different data visualization pages and I tried ChartJS, Plotly.js, and Chartist before I found apexcharts, and I haven't looked back. Apex had a better development experience than the other 3, partially due to the extremely well done documentation. Cheers!
